glib/src/lib.rs

```diff
@@ -64,11 +56,11 @@
 //! reference counted structures so the bindings have to implement corresponding
 //! smart pointers (wrappers), which encapsulate resource management and safety
 //! checks. Such wrappers are defined via the
-//! [`wrapper!`](macro.wrapper!.html) macro, which uses abstractions
-//! defined in the [`wrapper`](wrapper/index.html), [`boxed`](boxed/index.html),
-//! [`shared`](shared/index.html) and [`object`](object/index.html) modules.
+//! [`macro@wrapper`] macro, which uses abstractions
```

Comment:
Just wondering about this: do you want to keep the "macro@" (and "mod@" following) in the doc display?

Reply:
They aren't visible in the docs. Or what do you mean?

![Screenshot from 2021-04-11 13-19-33](https://user-images.githubusercontent.com/3466497/114302292-b83bde00-9ac8-11eb-99e2-01a623841fef.png)

Reply:
Oh nice. Then nevermind my comment. :)